app/main.py

```diff
@@ -1,34 +1,49 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self, start: tuple,
+                 end: tuple, is_drowned: bool = False) -> None:
```

Comment:
add more descriptive annotations, like tuple[int]. Fix it everywhere

Reply:
done